manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "KabarVirus.com",
+    "short_name": "Kabar Virus",
+    "start_url": "/?utm_source=homescreen",
+    "lang": "id",
+    "display": "standalone",
```

Comment:
I think it has to be `minimal-ui` since there are some possible back/forth navigations.